nj-core/src/convert.rs

```diff
@@ -62,12 +62,12 @@ impl TryIntoJs for () {
 impl<T, E> TryIntoJs for Result<T, E>
 where
     T: TryIntoJs,
-    E: ToString,
+    E: TryIntoJs,
 {
     fn try_to_js(self, js_env: &JsEnv) -> Result<napi_value, NjError> {
         match self {
             Ok(val) => val.try_to_js(&js_env),
-            Err(err) => Err(NjError::Other(err.to_string())),
+            Err(err) => Err(NjError::Native(err.try_to_js(&js_env)?)),
```

Comment:
I'll have to add `TryIntoJs` for `NjError`

Reply:
Are you able to do this? I'm running into this error when building the examples

```
error[E0599]: the method `try_to_js` exists for enum `Result<i32, NjError>`, but its trait bounds were not satisfied
   --> function/src/lib.rs:15:1
    |
15  | #[node_bindgen]
    | ^^^^^^^^^^^^^^^ method cannot be called on `Result<i32, NjError>` due to unsatisfied trait bounds
    |
   ::: /Users/nick/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/src/rust/library/core/src/result.rs:241:1
    |
241 | pub enum Result<T, E> {
    | --------------------- doesn't satisfy `Result<i32, NjError>: TryIntoJs`
    |
   ::: /Users/nick/infinyon/node-bindgen/nj-core/src/error.rs:12:1
    |
12  | pub enum NjError {
    | ---------------- doesn't satisfy `NjError: TryIntoJs`
    |
    = note: the following trait bounds were not satisfied:
            `NjError: TryIntoJs`
            which is required by `Result<i32, NjError>: TryIntoJs`
    = note: this error originates in an attribute macro (in Nightly builds, run with -Z macro-backtrace for more info)

error: aborting due to previous error

For more information about this error, try `rustc --explain E0599`.
error: could not compile `nj-example-function`
```

Reply:
Yeah, not yet, I'll update the PR tomorrow.